File: Some_Important_Topics/GCD.js
let a=24, b=60;
var divisor, divident, reminder;

if(a>b){
    divident = a;
    divisor = b;
}else{
    divident = b;
    divisor = a;
}

while(divisor!=0){
    reminder = divident%divisor;
    divident = divisor;
    divisor = reminder;
}

console.log(`Final Result : ${divident}`);

File: Some_Important_Topics/GCD.py
#  GCD = Greatest Common Divisor !

a,b = map(int,input("Enter 2 numbers : ").split(' '))
divisor, divident, reminder = 0,0,0
if a>b : 
  divident = a
  divisor = b
else :
  divident = b
  divisor = a
while divisor!=0 : 
  reminder = divident % divisor
  divident = divisor
  divisor = reminder
print(f"Final Result : {divident}")